test/es2015/symbol-unscopables.js

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2019-present Samsung Electronics Co., Ltd. and other contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Copyright 2014 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+
+object1 = {
+ property1: 42,
+ property2: 43
+};
+
+object1[Symbol.unscopables] = {
+ property1: true,
+ property2: false
+};
+try {
+    with (object1) {
+     print(property1);
+     // expected output: Error: property1 is not defined
+    }
+
```

Comment:
Please add a check code at this point that confirms an exception is occured above, such as inserting `assert(false)` at here. If exception does not occurred, this code will be excuted as normal.